poll/public/html/poll_edit.html

```diff
@@ -0,0 +1,96 @@
+{{js_template|safe}}
+<div class="wrapper-comp-settings is-active editor-with-buttons" id="settings-tab">
+    <form id="poll-form">
+    <ul class="list-input settings-list" id="poll-line-items">
+        <li class="field comp-setting-entry is-set">
+            <div class="wrapper-comp-setting">
+                <label class="label setting-label poll-setting-label" for="poll-display-name">Display Name</label>
+                <!-- In the case of surveys, this field will actually be used for block_name. -->
+                <input class="input setting-input" name="display_name" id="poll-display-name" value="{{ display_name }}" type="text" />
+            </div>
+        </li>
+        {% if not multiquestion %}
+            <li class="field comp-setting-entry is-set">
+              <h2><label for="poll-question-editor">Question/Prompt</label></h2>
+              <a href="//daringfireball.net/projects/markdown/syntax" target="_blank">Markdown Syntax</a> is supported.
+              <div id="poll-question-editor-container">
+                <textarea class="input setting-input" name="question" id="poll-question-editor">{{question}}</textarea>
+              </div>
+              <span class="tip setting-help">Enter the prompt for the user.</span>
+            </li>
+        {% endif %}
+        <li class="field comp-setting-entry is-set">
+           <h2><label for="poll-feedback-editor">Feedback</label></h2>
+           <a href="//daringfireball.net/projects/markdown/syntax" target="_blank">Markdown Syntax</a> is supported.
+           <div id="poll-feedback-editor-container">
+             <textarea class="input setting-input" name="feedback" id="poll-feedback-editor">{{feedback}}</textarea>
```

Comment:
Please put an ```id``` on the "tip" ```<span>``` below and reference it using an ```aria-describedby="[ID]"``` attribute on this form field